### PR TITLE
[DB Model] Database content schema initialization

### PR DESCRIPTION
**Pull request by MichaelCherry**
# Description

Initial creation of models and migration scripts.
NOTE: Models must all be within a file called models.py in a module included in INSTALLED_APPS, otherwise they will not be detected.

Migration 0002 should be applied automatically when you start docker.

## Type of change

- New feature (non-breaking change which adds functionality)

## Screenshot

N/A

# Testing

Nothing really atm, this is just model creation

# Checklist

Applicable for all code changes.

- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new compiler warnings
- [x] My changes generate no new accessibility errors and/or warnings
~~- [ ] I have added tests that prove my fix is effective or that my feature works~~
~~- [ ] I have researched WCAG2.1 accessibility standards and met them in this PR (can be navigated using a keyboard)~~
~~- [ ] My changes look good on IE 11+ and Chrome~~

### CODE REVIEW

File: backend/custom_models/sample_emib.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+[
```
Comment (fnormand01) [inline backend/custom_models/sample_emib.json]: Why do we need this file? For testing?

**Comment by fnormand01** [comment]
Do you have the database structure (schema or tables)? It could be great to add it to this PR in order to compare it with you new models.

File: backend/custom_models/sample_emib.json (hunk 1 above)
Comment (MichaelCherry) [reply backend/custom_models/sample_emib.json]: shoot.... didn't mean to push that one.......

Comment (MichaelCherry) [reply backend/custom_models/sample_emib.json]: Dropped in d3cc59c

**Comment by fnormand01** [comment]
Also, we'll have to figure out how to create models with different names than _models.py_.
Not necessary in this PR, but we have to fix that.

File: backend/custom_models/item.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from .item_type import ItemType
+
+
```
Comment (fnormand01) [inline backend/custom_models/item.py]: Can you just add simple comments in order to explain what's the goal/role of the model?
Please do the same for all the other models you added.

File: backend/custom_models/models.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+from .language import Language
+from .item_type import ItemType
+from .item import Item
+from .item_text import ItemText
+from .question_type import QuestionType
+from .question import Question
+
+# These imports help to audo discover the models
```
Comment (fnormand01) [inline backend/custom_models/models.py]: Little typo here: _audo_ instead of _auto_

File: backend/custom_models/item.py (hunk 2 above)
Comment (MichaelCherry) [reply backend/custom_models/item.py]: Added in dd37aae

File: backend/custom_models/models.py (hunk 3 above)
Comment (MichaelCherry) [reply backend/custom_models/models.py]: Fixed in dd37aae

**Pull request merged**